docker/build

```diff
@@ -11,6 +11,11 @@ set -e
 rm -rf docker/rvt/rvt-patch-llvm
 cp -r `pwd`/rvt-patch-llvm docker/rvt/rvt-patch-llvm
 
+# Copy cargo-verify over
+(cd cargo-verify; cargo clean)
+rm -rf docker/rvt/cargo-verify
+cp -r `pwd`/cargo-verify docker/rvt/cargo-verify
```

Comment:
(don't have to change) `cp -a` is more appropriate.

Reply:
Done